src/test/java/core/web/DeleteDashboardTest.java

```diff
@@ -0,0 +1,54 @@
+package core.web;
+
+import com.github.javafaker.Faker;
+import core.api.DashboardApiTests;
+import core.driver.RunType;
+import core.driver.WebDriverHolder;
+import core.model.User;
+import core.web.pageObjects.DashboardPage;
+import core.web.pageObjects.LoginPage;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+import java.util.List;
+
+public class DeleteDashboardTest extends BaseTest {
+
+    @Test
+    @org.testng.annotations.Test
+    public void deleteDashboardTest() {
+        final String homePageURL = webConfiguration.getRunType() == RunType.LOCAL ?
```

Comment:
it shouldn't be a part of the each test. Url should be resolved on the base page level